tests/route/test_static_route.py

```diff
@@ -118,33 +118,79 @@ def check_route_redistribution(duthost, prefix, ipv6, removed=False):
             assert prefix in adv_routes
 
 
-def run_static_route_test(duthost, ptfadapter, ptfhost, tbinfo, prefix, nexthop_addrs, prefix_len, nexthop_devs, ipv6=False, config_reload_test=False):
+def run_static_route_test(duthost, ptfadapter, ptfhost, tbinfo, prefix, nexthop_addrs, prefix_len, nexthop_devs, vlan_intf_addr, vlan_intf_devname,ipv6=False, config_reload_test=False):
     # Add ipaddresses in ptf
     add_ipaddr(ptfhost, nexthop_addrs, prefix_len, nexthop_devs, ipv6=ipv6)
 
     try:
+        ip_dst = str(ipaddress.ip_network(unicode(prefix))[1])
         # Add static route
         duthost.shell("sonic-db-cli CONFIG_DB hmset 'STATIC_ROUTE|{}' nexthop {}".format(prefix, ",".join(nexthop_addrs)))
         time.sleep(5)
 
-        # Check traffic get forwarded to the nexthop
-        ip_dst = str(ipaddress.ip_network(unicode(prefix))[1])
-        generate_and_verify_traffic(duthost, ptfadapter, tbinfo, ip_dst, nexthop_devs, ipv6=ipv6)
-
-        # Check the route is advertised to the neighbors
-        check_route_redistribution(duthost, prefix, ipv6)
-
```

Comment:
I would prefer not to remove this part, since we may only check for reachability after 'config reload'. I think we also need to make sure that the static route does its job before 'config reload'